drivers/scheduler/k8s/k8s.go

```
@@ -1461,6 +1465,46 @@ func (k *k8s) StartSchedOnNode(n node.Node) error {
 	return nil
 }
 
+// CreateCRDObjects and Validate their deployment
+func (k *k8s) CreateCRDObjects(ctx *scheduler.Context, timeout, retryInterval time.Duration) error {
+	var err error
+	k8sOps := k8s_ops.Instance()
+	for _, specObj := range ctx.App.SpecList {
+		if obj, ok := specObj.(*stork_api.ClusterPair); ok {
+			logrus.Info("Applying clusterpair specs")
+			err = k8sOps.CreateClusterPair(obj)
+			if err != nil {
+				logrus.Info("Error Creating Cluster Pair", err)
```

Infof() and need to add %v

---

Need to return err here instead of continuing forward.